src/reducers/Bluetooth/index.js

```diff
@@ -1,5 +1,10 @@
 import { combineReducers } from 'redux';
 import { BlinkReducer } from './BlinkReducer';
 import { ScanReducer } from './ScanReducer';
+import { UpdateReducer } from './UpdateReducer';
 
-export const BluetoothReducer = combineReducers({ blink: BlinkReducer, scan: ScanReducer });
+export const BluetoothReducer = combineReducers({
+  blink: BlinkReducer,
+  scan: ScanReducer,
+  update: UpdateReducer,
```

Comment:
Yeah.. I dunno what it is. I guess `blink` and `scan` are pretty generic also and they seem fine, and `update` under a `bluetooth` root reducer seems fine- also surely `UpdateActions.setLogInterval` gives enough context ? So I'm not sure what my problem is?